cost/google/schedule_instance/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.1
+
+- modified timezones_list.json file and optimized the code
```

Comment:
As someone who might be actually using this Policy, this ChangeLog entry is very vague and does not really convey much. What was the purpose of the `timezones_list.json` change? Did it add new timezones? What was optimized in the code?

This is falling into the trap of "bug fixes and performance improvements" which is not very helpful.

Reply:
updated the changelog with details